### PR TITLE
[Remove Stream/Optional usage] LPS-171238 frontend-taglib

### PR DESCRIPTION
**Pull request by JanisZhang**
Remove usages of Stream on Method Return Value for module: `frontend-taglib.`

https://issues.liferay.com/browse/LPS-171238

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:help

File: modules/apps/frontend-taglib/frontend-taglib-form-navigator/src/main/java/com/liferay/frontend/taglib/form/navigator/internal/configuration/FormNavigatorEntryConfigurationRetriever.java [hunk 1]
```diff
@@ -32,31 +32,29 @@
...
+	public List<String> getFormNavigatorEntryKeys(
 		String formNavigatorId, String categoryKey, String context) {
 
 		List<FormNavigatorEntryConfigurationParser>
 			formNavigatorEntryConfigurationParsers = ListUtil.fromCollection(
 				_serviceTrackerMap.getService(formNavigatorId));
 
-		Optional<List<String>> formNavigatorEntryKeysOptional =
-			Optional.empty();
+		List<String> formNavigatorEntryKeys = new ArrayList<>();
```
Comment (dantewang) [inline modules/apps/frontend-taglib/frontend-taglib-form-navigator/src/main/java/com/liferay/frontend/taglib/form/navigator/internal/configuration/FormNavigatorEntryConfigurationRetriever.java:42]: @JanisZhang The original default was `Optional.empty()`, which should be mapped to `null`, not any real object.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/frontend-taglib/frontend-taglib-form-navigator/src/main/java/com/liferay/frontend/taglib/form/navigator/internal/configuration/FormNavigatorEntryConfigurationRetriever.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/frontend-taglib/frontend-taglib-form-navigator/src/main/java/com/liferay/frontend/taglib/form/navigator/internal/configuration/FormNavigatorEntryConfigurationRetriever.java]: Hi @dantewang, I force-pushed and added null-check in its usages. Thanks!

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@JanisZhang failures not related.

**Comment by JanisZhang** [comment]
Thanks @catarineaguiar !

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/127977

**Pull request closed without merging**